content/what-is/top-iac-tools.md

```diff
@@ -0,0 +1,129 @@
+
+- **Flexibility**: Uses JSON-based templates or Bicep - a DSL. Lacks some of the abstraction and modularity of other tools.
+- **Integration**: Integrates well with Azure services. Broader integration to non-Azure CI/CD is more challenging.
+- **Ecosystem**: Supported by documentation, tutorials and troubleshooting advice and by the greater Azure ecosystem.
+
+### Google Cloud Deployment Manager
+
+Google Cloud Deployment Manager (CDM) focuses on managing Google Cloud Platform (GCP) resources using YAML and Python Jinja2 templates. CDM integrates deeply with Google Cloud and, like other native cloud tools, manages state internally. CDM offers resource grouping for enhanced resource management and cost management tools for expense tracking.
+
+While Google Cloud Deployment Manager's deep GCP integration is a strength, it faces challenges, including intricate template management and steep learning curves. Like other provider-specific solutions, CDM is ideal for teams with a strong focus on and investment in the Google Cloud ecosystem.
+
+- **Flexibility**: YAML or Jinja2 templating.
+- **Integration**: Integrates deeply with Google Cloud Platform (GCP) services. Additional features for integration with CI/CD pipelines.
+- **Ecosystem**: Supported by documentation and a smaller GCP-focused community.
+
+## Conclusion
+
+Choosing the right Infrastructure as Code tool is crucial in today's complex cloud landscape. We've explored options ranging from multi-cloud solutions to cloud-specific tools, each with unique strengths. The ideal choice depends on your needs, considering flexibility, multi-cloud support, state management, workflow integration, and ecosystem robustness.
+
+Each tool offers unique strengths and potential trade-offs:
+
+1. Multi-cloud tools like Pulumi, Terraform, and OpenTofu provide flexibility across cloud providers.
+
+2. Cloud-specific tools like AWS CloudFormation, Azure Resource Manager, and Google Cloud Deployment Manager offer deep integration with their respective platforms.
+
+We've shown you the options – now let us make our case for Pulumi. Pulumi's use of general-purpose programming languages for infrastructure definition provides familiar syntax, powerful abstractions, and seamless integration with existing development workflows. It's really something you need to experience.
```

Comment:
```suggestion
```

The case should have been made in the Pulumi section in my opinion.

Reply:
I like having the more clearly biased part at the end, where sections are more rubric based, but might just be my own taste.